linkouts/linkouts/__main__.py

```diff
@@ -40,13 +49,37 @@ def parseArgs():
     version=f'%(prog)s {linkouts.__version__}',
   )
 
+  # logging args
+  loglevel_envvar = 'LOG_LEVEL'
```

Comment:
The log level and log file info isn't actually used to configure the Python logging module!